src/controllers/initgamecontroller.js

```diff
@@ -29,6 +29,10 @@ class InitGameController extends Controller
                 sim.registerEvent();
             }
         );
+
+        $(document).ready(function () {
```

Comment:
I can't get that to work. 
It gives the error `Your tripData is not valid at index: 0`. I think the views have not loaded properly. And therefore Trip.js cannot find the selected elements.